src/widgets/numeric-menu/defaultTemplates.js

```diff
@@ -1,8 +1,7 @@
 /* eslint-disable max-len */
 export default {
-  header: '',
   item: `<label class="{{cssClasses.label}}">
-  <input type="radio" class="{{cssClasses.radio}}" name="{{attributeName}}" {{#isRefined}}checked{{/isRefined}} />{{label}}
+  <input type="radio" class="{{cssClasses.radio}}" name="{{attribute}}" {{#isRefined}}checked{{/isRefined}} />
```

Comment:
The `attribute` value is not provided to the template so we still have `undefined`.

```html
<input type="radio" class="ais-NumericMenu-radio" name="">
```